### PR TITLE
update image_visualize.py

### PR DESCRIPTION
**Pull request by Sibasish-Padhy**

### CODE REVIEW

File: klar_eda/visualize/image_visualize.py [hunk 1]
```diff
@@ -24,6 +24,11 @@
 ############################################################################
 
 class ImageDataVisualize:
+# the first function is used intialize/get data in the form of images and labels.
```
Comment (Ask149) [inline klar_eda/visualize/image_visualize.py:27]: Constructors need not be commented, with such details. Mention the parameters and a one-line description of each of the parameters.

File: klar_eda/visualize/image_visualize.py [hunk 2]
```diff
@@ -48,7 +53,13 @@ def __init__(self, data, labels, boxes=None):
         })
         self.dataset['area'] = self.dataset['Height'] * self.dataset['Width']
         print('Number of images after validation and filtering:', self.num_images)
-
+#This function is written for saving the file(of the program) for better access and smooth running.
```
Comment (Ask149) [inline klar_eda/visualize/image_visualize.py:56]: Keep the description limited to 3 lines and then describe the method parameters individually on a separate line.

**Review by Ask149 — CHANGES_REQUESTED** [review]
Hi @Sibasish-Padhy, Please refer to the [preprocessing.py](https://github.com/klarEDA/klar-EDA/blob/master/klar_eda/preprocessing.py) or the [Spnix documentation](https://sphinx-rtd-tutorial.readthedocs.io/en/latest/docstrings.html) for the format of the method documentation.

File: klar_eda/visualize/image_visualize.py (hunk 2 above)
Comment (Sibasish-Padhy) [reply klar_eda/visualize/image_visualize.py:56]: thank you for your feedback sir will surely improve upon it

Comment (Sibasish-Padhy) [reply klar_eda/visualize/image_visualize.py:56]: any more anomalies detected sir??

**Pull request closed without merging**